ee/app/livechat-enterprise/server/settings.js

```diff
@@ -2,57 +2,6 @@ import { settings } from '../../../../app/settings';
 import { Settings } from '../../../../app/models/server';
 
 export const createSettings = () => {
-	settings.add('Livechat_waiting_queue', false, {
```

Comment:
Do these changes demand a migration or just changing the setting section is enough?

Reply:
I tested locally, believing a migration would be needed, but seems like thoseind of updates are done on startup, so no need to migrate